packages/errors/tsconfig.json

```diff
@@ -4,6 +4,7 @@
         "lib": [
             "DOM",
             "ES2015",
+            "ES2022.Error"
```

Comment:
Does this mean people downstream would have to include this in their `tsconfig.json` as well?

Reply:
Yes, but if they don't have it in their tsconfig already then they're not missing it (ie. they don't have `cause` in their typespec for _any_ `Errors`).